<ui version="4.0" >
 <class>DotTelDialog</class>
 <widget class="QDialog" name="DotTelDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>.tel Dial Mode</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>.tel Information</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLabel" name="infoTextLabel" >
     <property name="text" >
      <string>Company information</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QListWidget" name="numbersList" />
   </item>
   <item row="3" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>121</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="cancelButton" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QPushButton" name="dialButton" >
     <property name="text" >
      <string>Dial Selected Number</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
